pages/common/diskonaut.md

```diff
@@ -0,0 +1,20 @@
+# diskonaut
+
+> TUI disk space navigator written in Rust.
```

Comment:
I suggest to avoid acronyms and to remove implement details. What about:
```suggestion
> Terminal disk space navigator.
```

Reply:
I agree with the acronym, but why remove the language?

Reply:
A simple `rg -i 'written in'` reveals:
```
common/spike.md:> A fully featured static website generator written in JavaScript.
common/loc.md:> Tool written in Rust that counts lines of code.
common/middleman.md:> Static site generator written in Ruby.
common/ect.md:> File optimizer written in C++. It supports `.png`, `.jpg`, `.gzip` and `.zip` files.
common/gocryptfs.md:> Encrypted overlay filesystem written in Go.
common/espanso.md:> Cross-platform Text Expander written in Rust.
common/rspec.md:> Behavior-driven development testing framework written in Ruby to test Ruby code.
common/xsv.md:> A CSV command line toolkit written in Rust.
common/androguard.md:> Reverse engineering tool for Android applications. Written in Python.
common/scc.md:> Tool written in Go that counts lines of code.
common/gopass.md:> Standard Unix Password Manager for Teams. Written in Go.
common/tmpmail.md:> A temporary email right from your terminal written in POSIX sh.
common/chezmoi.md:> A multi-machine dotfile manager, written in Go.
common/rails.md:> A server-side MVC framework written in Ruby.
```
And personly, I like to know what it's written in  (especially if it's Rust or Go), as long as it doesn't push more valuable information out of the way (as slightly seems the case with `ect`).

Reply:
My initial sentence is also directly from `diskonaut`'s description (AUR)

Reply:
I like this the best. What do you think, @nicokosi & @Waples?
```suggestion
> Terminal disk space navigator, written in Rust.
```

Reply:
Oops, I'll put that in a new suggestion...